recipes/default.rb

```diff
@@ -4,6 +4,11 @@
 #
 # Copyright:: 2018, The Authors, All Rights Reserved.
 
+return if node[cookbook_name]['zookeeper_hosts'].empty?
```

Comment:
@ajeygore here is the checks for empty value...

Reply:
oh good! missed.